src/gobimport/__main__.py

```diff
@@ -42,7 +45,17 @@ def extract_dataset_from_msg(msg):
     return get_import_definition(header['catalogue'], header['collection'], header.get('application'))
 
 
-def handle_import_msg(msg):
+def handle_import_msg(msg: dict, use_message_broker: bool = True, destination: str = None) -> dict:
```

Comment:
I would extract this into two separate functions. One for the "import message" and the other one for the "standalone" run. Not that the current way is wrong, but purely from an abstraction point of view.

- "handle_import_msg" is a very descriptive name for a message handler, not so much for when this code is run standalone. You would not expect this code to be executed when running as standalone.
- Separating into two functions prevents the "use_message_broker" parameter.
- It prevents the creation of the artificial "msg" in the main() function. This can be done in the run_as_standalone() function.
- This keeps the main() function lean

Now the abstractions are much clearer:
- handle_import_msg just does wat it says: handling a message
- run_as_standalone does exactly this. It still maps the args to a msg, but at least that is contained within this function. Just pass all the args to this function and do the transformation to msg here.
- main() is easier to read and has only one function: choosing between a handler or standalone execution